Upstream-Status: Submitted [https://crrev.com/c/1454356]

* Submitted to upstream, waiting approval

Signed-off-by: Maksim Sisov <msisov@igalia.com>
---
From 028f18f7dacc13cad92d01309682e346a64baa16 Mon Sep 17 00:00:00 2001
From: Maksim Sisov <msisov@igalia.com>
Date: Tue, 5 Feb 2019 08:29:16 +0200
Subject: [PATCH] Add missing algorithm header to webcursor_ozone

This fixes the following error:

../../content/common/cursors/webcursor_ozone.cc:59:77: error: no matching function for call to 'min(<brace-enclosed initializer list>)'
        static_cast<float>(maximum_cursor_size_.height()) / bitmap->height()});
                                                                             ^
Bug: 819294

Change-Id: I248e8b0199d150bb22cf0609c3588048e6405bfe
---
 content/common/cursors/webcursor_ozone.cc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/content/common/cursors/webcursor_ozone.cc b/content/common/cursors/webcursor_ozone.cc
index 59f880a327b0..c9b9cb1d36ef 100644
--- a/content/common/cursors/webcursor_ozone.cc
+++ b/content/common/cursors/webcursor_ozone.cc
@@ -4,6 +4,8 @@
 
 #include "content/common/cursors/webcursor.h"
 
+#include <algorithm>
+
 #include "third_party/blink/public/platform/web_cursor_info.h"
 #include "ui/base/cursor/cursor.h"
 #include "ui/base/cursor/cursor_util.h"
-- 
2.17.1

